lib/diego/client.rb

```diff
@@ -180,8 +158,15 @@ def protobuf_encode!(hash, protobuf_message_class)
       raise EncodeError.new(e.message)
     end
 
-    def validate_status!(response:, statuses:)
-      raise ResponseError.new("failed with status: #{response.status}, body: #{response.body}") unless statuses.include?(response.status)
+    def request(body: nil, path:)
```

Comment:
I think `post_request` would be a better name.

Reply:
Agreed